etc/kayobe/inventory/group_vars/monitoring/firewall

```diff
@@ -8,6 +8,11 @@ firewallgen_ipv4_input_allow_rules:
     proto: tcp
     destination: "127.0.0.1"
     comment: "hint: used by 'master'"
+  - interface: "lo"
+    port: 24224
```

Comment:
Good spot, was going to ask about whether this was included. This is where caso send data to.